DynamicSmoothedGUI.py

```diff
@@ -134,51 +134,46 @@ def visibilityDraw(self):
         upper_row = int(min(row + index_radius_outer, self.gridFull.num_rows))
         upper_col = int(min(col + index_radius_outer, self.gridFull.num_cols))
```

Comment:
why do these need to be ints?

Reply:
ah good catch, you're right, those are already guaranteed to be ints